### ISSUE TITLE
Country code repeated twice when enter without + sign

### ISSUE THREAD

**Issue by pavawit** [catamphetamine/libphonenumber-js#365]
When entering number country code has + sign
(+66)89-900-9000 It perfectly returns  +66899009000

But (66)89-900-9000 it repeated the country code twice

```javascript
var libphonenumberJs = require("libphonenumber-js/max")

var phoneNumber = libphonenumberJs.parsePhoneNumberFromString("(+66)89-900-9000", 'TH')
console.log(phoneNumber.number) // "+66899009000"

var phoneNumber2 = libphonenumberJs.parsePhoneNumberFromString("(66)89-900-9000", 'TH')
console.log(phoneNumber2.number) // "+6666899009000"
```

**Comment by catamphetamine** [comment]
This could be related to this issue: https://github.com/catamphetamine/libphonenumber-js/issues/318
I'm planning to look into it soon.

**Comment by catamphetamine** [comment]
Fixed in `libphonenumber-js@1.7.32`.
